### PR TITLE
Allow us to set a custom security repo.

### PR DESCRIPTION
**Pull request by dennisse**
This might be useful in enterprise environments, and will be usefull for
Raspberry PIs, which use custom security repos.

### CODE REVIEW

**Comment by dennisse** [comment]
Amended to include a change of the README.

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Manage packages and up(date|grade)s in Debian-like systems.
 * `apt_ubuntu_partner_enable`: [default: `false`]: Whether or not to enable the `partner` repository
 * `apt_ubuntu_extras_enable`: [default: `false`]: Whether or not to enable the `extras` repository (only applies to < 16.04)
 * `apt_debian_mirror`: [default: `http://deb.debian.org/debian/`]: The mirror to use
+* `apt_debian_security_mirror`: [default: `http://security.ubuntu.com/ubuntu`]: The security-mirror to use
```
Comment (tersmitten) [inline README.md:24]: `apt_ubuntu_security_mirror`?

**Comment by tersmitten** [comment]
Looks like a nice addition

File: README.md (hunk 1 above)
Comment (dennisse) [reply README.md:24]: Woops :)

**Comment by dennisse** [comment]
I ammended the PR to include security-repo for debian.

Sorry about the force-pushing.

**Pull request merged**